stetho/src/main/java/com/facebook/stetho/inspector/domstorage/SharedPreferencesHelper.java

```diff
@@ -0,0 +1,39 @@
+package com.facebook.stetho.inspector.domstorage;
+
+import android.content.Context;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SharedPreferencesHelper {
+  private static final String PREFS_SUFFIX = ".xml";
+
+  private SharedPreferencesHelper() {
+  }
+
+  public static List<String> getSharedPreferenceTags(Context context) {
+    ArrayList<String> tags = new ArrayList<String>();
+
+    String rootPath = context.getApplicationInfo().dataDir + "/shared_prefs";
+    File root = new File(rootPath);
+    if (root.exists()) {
+      for (File file : root.listFiles()) {
```

Comment:
In the scheme of things we're doing in Stetho, I'm not so sure this is our riskiest assumption.... :)